packages/firestore/src/local/simple_db.ts

```diff
@@ -158,6 +158,7 @@ export class SimpleDbTransaction {
  */
 export class SimpleDb {
   private db?: IDBDatabase;
+  private lastDbVersion: number | null = null;
```

Comment:
you might want a comment on this line that the value should only be set on db close otherwise it should be null. That's not completely clear from the name alone.

Reply:
Hmm, good point. I renamed the variable from `lastDbVersion` to `lastClosedDbVersion` to address this. Do you think that is sufficient, or is an inline comment still needed in your opinion? Thanks!